### ISSUE TITLE
Anonymous session tokens? 

### ISSUE THREAD

**Issue by yochannah** [intermine/intermine-ws-python#21]
Does the Python client have any way to create anonymous session tokens? It could be useful for demo code where I didn't want to expose the token associated with my account and I didn't need the list operations I was performing to persist, long-term.

I couldn't see one, and ended up writing something like this (it's probably awful, given my python skills... ;)

```
service_url = "http://www.humanmine.org/humanmine/"
token = requests.get(url=service_url + "/service/session")
token = token.json()["token"]
service = Service(service_url, token=token)
```

**Comment by nupurgunwant** [comment]
Hi @yochannah , yes its possible but only for 24 hours. 
You are using the sessions tab of InterMine API in your code : "http://iodocs.apps.intermine.org/flymine/docs#/ws-session/GET/session"

Maybe we can make a function for this, say get_random_token() ?

**Comment by yochannah** [comment]
fixed as part of #47
